toolset/setup/linux/prerequisites.sh

```diff
@@ -62,6 +62,8 @@ RETCODE=$(fw_exists ~/.bash_profile.bak)
 }
 
 sudo sh -c "echo '*               -    nofile          65535' >> /etc/security/limits.conf"
+sudo sh -c "echo '*            hard    rtprio             99' >> /etc/security/limits.conf"
```

Comment:
This sounds completely reasonable, but I am WAY out of my depth here. Pinging @hamiltont and @bhauer to try and get a concensus.